docs/UserGuide.md

```diff
@@ -98,21 +98,16 @@ Adds a patient to the address book.
 
 Format: `add id/PATIENT_HOSPITAL_ID n/NAME p/PREFERRED_NAME f/FOOD_PREFERENCE+ c/FAMILY_CONDITION+ h/HOBBY+ [t/TAG]…​`
 
-<box type="tip" seamless>
-
-**Tip:** 
 * The `add` command accepts parameters which consists of:
-    *  `patientHospitalId` integer,
-    * `name`, `preferredName` String with only alphabets character,
+    *  `patientHospitalId` non-negative numeric characters which uniquely identifies the patient,
```

Comment:
maybe it might be better to say  non-negative numeric string instead as people might be wondering why not use non-negative integer?